app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,40 @@
+!!!
+%html
+  %head
+    %meta{:content => "text/html; charset=UTF-8", "http-equiv" => "Content-Type"}/
```

Comment:
Please use new syntax style, without "rocket"

Reply:
Same in lines 17, 18
And also with all "href"